Tools/TestWebKitAPI/Tests/WebKitCocoa/WebPushDaemon.mm

```diff
@@ -73,13 +73,20 @@ @interface PushNotificationDelegate : NSObject<WKUIDelegatePrivate, _WKWebsiteDa
 }
 -(void)clearMostRecents;
 
+@property BOOL expectsBuiltInNotificationCallbacks;
 @property (nonatomic, readonly) RetainPtr<_WKNotificationData> mostRecentNotification;
 @property (nonatomic, readonly) RetainPtr<NSURL> mostRecentActionURL;
 @property (nonatomic, readonly) std::optional<uint64_t> mostRecentAppBadge;
 @end
 
 @implementation PushNotificationDelegate
 
+-(id)init {
+    if (self = [super init])
+        self.expectsBuiltInNotificationCallbacks = YES;
```

Comment:
nit: this naming feels a bit confusing to me because when the builtInNotifications feature is on you actually expect builtInNotificationCallbacks to be false. Maybe rename to expectsDataStoreNotificationCallbacks or something?